Eastern-Research-Group/expert-query
# Feature/202 connect glossary terms

## Related Issues:
* [EQ-202](https://jira.epa.gov/browse/EQ-202)

## Main Changes:
* Updated the input field tooltips to draw from the plaintext glossary definitions.
* Fixed an issue with the `associatedActionStatus` field drawing from the wrong set of domain values.
* Fixed the _Contact Us_ link at the bottom of the page, which hadn't been updated.

## Steps To Test:
1. In the `etl` folder, run the task `npm run etl_glossary`.
2. Start the application, then go to the home page.
3. Test the different profiles, and verify that each filter field has an associated tooltip.
